src/main/java/net/zerowatch/SkillRatingCommand.java

```diff
@@ -1,21 +1,25 @@
 package net.zerowatch;
 
+import me.ramswaroop.jbot.core.slack.models.RichMessage;
+import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-import me.ramswaroop.jbot.core.slack.models.RichMessage;
 
 /**
  * Handles request for SR lookups.
  */
 @RestController
 public final class SkillRatingCommand {
 
-  @RequestMapping(value = "/test",
-      method = RequestMethod.GET)
+  @RequestMapping(value = "/skills",
```

Comment:
Is this was makes the command /skills on Slack? Or do we hook /sr to this Slack-side.